### PR TITLE
add karaoke

### PR DESCRIPTION
**Pull request by Shane4666**
@casey-collab

### CODE REVIEW

File: english/karaoke/nope.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+You walk into a bar and see karaoke is going on!
+
+[nope out](../marshmellow.md)
```
Comment (casey-collab) [inline english/karaoke/nope.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/karaoke/nope.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+
+[sing karaoke?](karaoke/karaoke/sing.md)
```
Comment (casey-collab) [inline english/karaoke/nope.md]: This link is also invalid.

File: english/karaoke/nope.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+You walk into a bar and see karaoke is going on!
```
Comment (casey-collab) [inline english/karaoke/nope.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by Shane4666** [comment]
@casey-collab

File: english/karaoke/nope.md (hunk 3 above)
Comment (casey-collab) [inline english/karaoke/nope.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by Shane4666** [comment]
@casey-collab

**Comment by casey-collab** [comment]
Looks great! :wind_chime:

**Pull request merged**